### ISSUE TITLE
Warning when creating a new calendar

### ISSUE THREAD

**Issue by YasserHassan** [quantopian/trading_calendars#50]
When creating a new calendar, a `FutureWarning` is generated. pandas version used is 0.24.1.

`us_calendar = get_calendar('XNYS')`

gives:

> /usr/lib/python3.7/site-packages/pandas/core/sorting.py:257: FutureWarning: Converting timezone-aware DatetimeArray to timezone-naive ndarray with 'datetime64[ns]' dtype. In the future, this will return an ndarray with 'object' dtype where each element is a 'pandas.Timestamp' with the correct 'tz'.
	To accept the future behavior, pass 'dtype=object'.
	To keep the old behavior, pass 'dtype="datetime64[ns]"'.
  items = np.asanyarray(items)
/usr/lib/python3.7/site-packages/pandas/core/sorting.py:257: FutureWarning: Converting timezone-aware DatetimeArray to timezone-naive ndarray with 'datetime64[ns]' dtype. In the future, this will return an ndarray with 'object' dtype where each element is a 'pandas.Timestamp' with the correct 'tz'.
	To accept the future behavior, pass 'dtype=object'.
	To keep the old behavior, pass 'dtype="datetime64[ns]"'.
  items = np.asanyarray(items)

**Comment by pando85** [comment]
It is giving an Error to me now:

```ipython
us_calendar = get_calendar('XNYS')                                                                                                                 
~/.pyenv/foo/lib/python3.7/site-packages/pandas/core/sorting.py:257: FutureWarning: Converting timezone-aware DatetimeArray to timezone-naive ndarray with 'datetime64[ns]' dtype. In the future, this will return an ndarray with 'object' dtype where each element is a 'pandas.Timestamp' with the correct 'tz'.
	To accept the future behavior, pass 'dtype=object'.
	To keep the old behavior, pass 'dtype="datetime64[ns]"'.
  items = np.asanyarray(items)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-38-c96578fd303a> in <module>
----> 1 us_calendar = get_calendar('XNYS')

~/.pyenv/foo/lib/python3.7/site-packages/trading_calendars/calendar_utils.py in get_calendar(self, name)
    145 
    146         # Cache the calendar for future use.
--> 147         calendar = self._calendars[canonical_name] = factory()
    148         return calendar
    149 

~/.pyenv/foo/lib/python3.7/site-packages/trading_calendars/trading_calendar.py in __init__(self, start, end)
    125         # the open/close time.
    126         _special_opens = self._calculate_special_opens(start, end)
--> 127         _special_closes = self._calculate_special_closes(start, end)
    128 
    129         # Overwrite the special opens and closes on top of the standard ones.

~/.pyenv/foo/lib/python3.7/site-packages/trading_calendars/trading_calendar.py in _calculate_special_closes(self, start, end)
    996             self.special_closes_adhoc,
    997             start,
--> 998             end,
    999         )
   1000 

~/.pyenv/foo/lib/python3.7/site-packages/trading_calendars/trading_calendar.py in _special_dates(self, calendars, ad_hoc_dates, start_date, end_date)
    963                 self.tz,
    964             )
--> 965             for time_, calendar in calendars
    966         ]
    967 

~/.pyenv/foo/lib/python3.7/site-packages/trading_calendars/trading_calendar.py in <listcomp>(.0)
    963                 self.tz,
    964             )
--> 965             for time_, calendar in calendars
    966         ]
    967 

~/.pyenv/foo/lib/python3.7/site-packages/trading_calendars/trading_calendar.py in scheduled_special_times(calendar, start, end, time, tz)
   1008     days = calendar.holidays(start, end)
   1009     return pd.Series(
-> 1010         index=pd.DatetimeIndex(days, tz='UTC'),
   1011         data=days_at_time(days, time, tz=tz),
   1012     )

~/.pyenv/foo/lib/python3.7/site-packages/pandas/core/indexes/datetimes.py in __new__(cls, data, freq, start, end, periods, tz, normalize, closed, ambiguous, dayfirst, yearfirst, dtype, copy, name, verify_integrity)
    301             data, dtype=dtype, copy=copy, tz=tz, freq=freq,
    302             dayfirst=dayfirst, yearfirst=yearfirst, ambiguous=ambiguous,
--> 303             int_as_wall_time=True)
    304 
    305         subarr = cls._simple_new(dtarr, name=name,

~/.pyenv/foo/lib/python3.7/site-packages/pandas/core/arrays/datetimes.py in _from_sequence(cls, data, dtype, copy, tz, freq, dayfirst, yearfirst, ambiguous, int_as_wall_time)
    374             data, dtype=dtype, copy=copy, tz=tz,
    375             dayfirst=dayfirst, yearfirst=yearfirst,
--> 376             ambiguous=ambiguous, int_as_wall_time=int_as_wall_time)
    377 
    378         freq, freq_infer = dtl.validate_inferred_freq(freq, inferred_freq,

~/.pyenv/foo/lib/python3.7/site-packages/pandas/core/arrays/datetimes.py in sequence_to_dt64ns(data, dtype, copy, tz, dayfirst, yearfirst, ambiguous, int_as_wall_time)
   1766     if is_datetime64tz_dtype(data):
   1767         # DatetimeArray -> ndarray
-> 1768         tz = maybe_infer_tz(tz, data.tz)
   1769         result = data._data
   1770 

~/.pyenv/foo/lib/python3.7/site-packages/pandas/core/arrays/datetimes.py in maybe_infer_tz(tz, inferred_tz)
   1973         raise TypeError('data is already tz-aware {inferred_tz}, unable to '
   1974                         'set specified tz: {tz}'
-> 1975                         .format(inferred_tz=inferred_tz, tz=tz))
   1976     return tz
   1977 

TypeError: data is already tz-aware UTC, unable to set specified tz: UTC

```

```bash
#python --version
Python 3.7.3

#pip freeze | grep trading-calendars
trading-calendars==1.7.0
```

**Comment by pando85** [comment]
Ok, this is fixed in  #48. For me it is resolved, it could be cool release a new version with this fix :)

**Comment by gerrymanoim** [comment]
This has been released now.
